README.md

```diff
@@ -374,6 +374,16 @@ And finally, create your consumer like so:
 App.cable = Cable.createConsumer()
 ```
 
+If you are using the same server to handling the websockets, you can create the connection with a relaitve path:
+
+```coffeescript
+# app/assets/javascripts/application_cable.coffee
+#= require cable
+
+@App = {}
+App.cable = Cable.createConsumer "/"
+```
+
```

Comment:
This section of the README describes using `<%= action_cable_meta_tag %>` so I don't think this belongs here. Maybe just add a sentence to the description following the first `Cable.createConsumer` example.